book/src/installation_windows.md

```diff
@@ -16,26 +16,39 @@ Install the rust toolchain via [rustup](https://rustup.rs/).
 
 ### Set Rust toolchain to MSVC
 
-Set the Rust toolchain to MSVC by executing:
+Tell Rust to use MSVC by executing:
 
 ```
 rustup default stable-msvc
 ```
 
-### Build GTK 4 
+Note that this does not install the MSVC toolchain. If you don't have it yet, you will need to [install it separately](https://learn.microsoft.com/en-us/windows/dev-environment/rust/setup).
+
+### Build GTK 4
+
+Follow the [`gvsbuild` docs to build GTK 4](https://github.com/wingtk/gvsbuild#development-environment). Be sure to build the right version:
 
-Follow the [`gvsbuild` docs to build GTK 4](https://github.com/wingtk/gvsbuild#development-environment).
-When choosing the GTK version to build, select `gtk4` instead of `gtk3`: 
 ```
 gvsbuild build gtk4
 ```
-<!--  -->
-### Update `Path` environment variable
 
-2. Add `New User Variable` and update `Path` in environment variable to include `PKG_CONFIG_PATH` and the GTK 4 libraries:
-   1. Go to settings -> Search and open `Advanced system settings` -> Click on `Environment variables`
-   2. Select `New` -> Input `Variable name` : `PKG_CONFIG_PATH` & `Variable value` : `C:\gtk-build\gtk\x64\release\lib\pkgconfig`
-   3. Select `Path` -> Click on `Edit` -> Add `C:\gtk-build\gtk\x64\release\bin`
+### Update environment variables
+
+1. Go to Start
+2. Search for 'Advanced system settings'
+3. Click 'Environment Variables...'
+4. Ensure there is a user variable named `PKG_CONFIG_PATH` with value `C:\gtk-build\gtk\x64\release\lib\pkgconfig`
+5. Edit the variable named `Path` and add `C:\gtk-build\gtk\x64\release\lib\pkgconfig` to it
+
+### Ensure your linker can find GTK's library files
+
+Open `%HOMEPATH%\.cargo\config` for editing. You can add extra flags for your linker here to tell it where GTK's library files are. If you're using LLVM's linker with the `stable-x86_64-pc-windows-msvc` Rust toolchain, your config may look like this:
+
+```toml
+[target.x86_64-pc-windows-msvc]
+linker = "lld-link.exe"
+rustflags = ["-C", "link-args=/LIBPATH:C:\\gtk-build\\gtk\\x64\\release\\lib\\"]
+```
```

Comment:
I assume that's only necessary if you use `lld`?
If yes, please make that clear here

Reply:
You also need to use it when using the default linker, I just didn't research the appropriate flag to do so. Here's what happens when I remove my `~/.cargo/config`:

```
error: linking with `link.exe` failed: exit code: 1181
(...)
  = note: LINK : fatal error LNK1181: cannot open input file 'gobject-2.0.lib'
```

This works fine for me:

```toml
[target.x86_64-pc-windows-msvc]
rustflags = ["-C", "link-args=/LIBPATH:C:\\gtk-build\\gtk\\x64\\release\\lib\\"]
```

It looks like both linkers support the same flag, so I'll update the documentation accordingly.

Reply:
I wonder why I didn't have to do that, but oh well.
Let's add it :)